backend/engine/linux_finder.py

```diff
@@ -1,19 +1,15 @@
 import os
+from xdg import XDG_DATA_HOME
 
-STEAM_DIRS = [
-    '/Steam',
-    '/bin/Steam',
-    '/usr/bin/Steam'
-] # TODO - correct?
-
+STEAM_DIR = 'Steam'
+SAVE_DIR = 'Paradox Interactive/Stellaris/save games'
 
 def find_steam():
-    for folder in STEAM_DIRS:
-        if os.path.isdir(folder): # TODO - use listdir and tolower for case insensitive
-            return path
+    path = os.path.join(str(XDG_DATA_HOME), STEAM_DIR)
```

Comment:
Is this where Steam typically gets installed on Linux? Never used it on Linux myself

Reply:
The default location for Steam is: `~/.local/share/Steam/`

Reply:
Thanks! I added the second location